api/src/main/kotlin/nl/avisi/anowalke/services/VertaalService.kt

```diff
@@ -0,0 +1,83 @@
+package nl.avisi.anowalke.services
+
+import com.google.gson.Gson
+import com.google.gson.reflect.TypeToken
+import mu.KotlinLogging
+import nl.avisi.anowalke.dto.ExpressionsDto
+import nl.avisi.anowalke.dto.Translations
+import nl.avisi.anowalke.util.configureObjectMapper
+import org.springframework.http.HttpEntity
+import org.springframework.http.HttpHeaders
+import org.springframework.http.ResponseEntity
+import org.springframework.http.converter.json.MappingJackson2HttpMessageConverter
+import org.springframework.stereotype.Component
+import org.springframework.web.client.HttpStatusCodeException
+import org.springframework.web.client.RestTemplate
+
+@Component
+class VertaalService {
+
+    val dummylist = listOf("subscription","key","region","endpoint","url","client","inputlist","request","body","media","type","gson","response","list","of","my","class","object","translations","it","post","json_text","parser","json","prettify","translator")
```

Comment:
Dit is nu ivm het niet werken van de call? Wat netter kan zijn is om een interface voor een connector aan te maken, iets als ITranslateConnector o.i.d. En vervolgens een implementatie aan te maken die doet wat jij eerst wilde doen met de translate service. En nog een andere implementatie die jouw dummy lijst terug geeft. @ConditionalOnProperty zou je dan kunnen gebruiken om één van de twee beans aan te laten maken. Is ook voor de toekomst makkelijker om dan van vertaalservice te wisselen. Bijvoorbeeld als de ene het opeens niet meer doet ;)

Reply:
Ohhhh slim c: